### ISSUE TITLE
-Wunused-function in arch/arm/mach-omap1/board-ams-delta.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#1326]
via omap1_defconfig on linux-next:
```
arch/arm/mach-omap1/board-ams-delta.c:462:12: warning: unused function 'ams_delta_camera_power' [-Wunused-function]
static int ams_delta_camera_power(struct device *dev, int power)
           ^
```

**Comment by nickdesaulniers** [comment]
Submitted: https://lore.kernel.org/lkml/20210401160434.7655-1-maciej.falkowski9@gmail.com/

**Comment by m-falkowski** [comment]
Patch accepted: https://lore.kernel.org/lkml/YK4rDeJSS9rEAF0W@atomide.com/

**Comment by nickdesaulniers** [comment]
fixed: bae989c4bc53f861cc1b706aab0194703e9907a8
